lib/scsi_defs.h

```diff
@@ -97,5 +99,77 @@ namespace scsi_defs {
   } ABSL_ATTRIBUTE_PACKED;
   static_assert(sizeof(InquiryCommand) == kInquiryCommandSize);
 
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6.2 Table 60
+  enum class PeripheralQualifier: uint8_t {
+    kPeripheralDeviceConnected = 0b000,
+    kPeripheralDeviceNotConnected = 0b001,
+    kReserved = 0b010,
+    kNotSupported = 0b011
+  };
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6.2 Table 63
+  enum class TPGS : uint8_t {
+    kNotSupported = 0b00,
+    kImplicitAccess = 0b01,
+    kExcplicitAccess = 0b10,
+    kFullAccess = 0b11,
+  };
+
+  enum class ResponseDataFormat : uint8_t {
+    kObsolete0 = 0x0,
+    kObsolete1 = 0x1,
+    kCompliant = 0x2,
+  };
+
+  // Refer to https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf , Section 3.6.2 Table 59
+  // Fields are arranged according to Big Endian format
+  struct InquiryData {
+    PeripheralQualifier peripheral_qualifier: 3;
+    PeripheralDeviceType peripheral_device_type: 5;
+    bool rmb: 1; // Removable Media Bit (RMB)
+    bool lu_cong: 1;
+    uint8_t reserved_1: 6;
+    Version version: 8;
+    bool reserved_2: 2;
+    bool normaca: 1; //Normal ACA (NORMACA)
+    bool hisup: 1; // Hierarchical Support Bit (HISUP)
+    ResponseDataFormat response_data_format: 4;
+    uint8_t additional_length: 8;
+    bool sccs: 1; // SCC Supported
+    bool acc: 1; // Access controls Coordinator Bit
+    TPGS tpgs: 2; // Target Port Group Support (TPGS)
+    bool third_party_copy: 1; // referred to as 3PC in the documentation
+    uint8_t reserved_3: 2;
+    bool protect: 1;
+    bool obsolete_1: 1;
+    bool encserv: 1; // Enclosure Services Bit
+    bool vs_1: 1; // vendor specific bit
+    bool multip: 1; // multiple SCSI Port
+    bool obsolete_2: 1;
+    bool reserved_4: 2;
+    bool addr_16: 1; // SCSI 16-bit address support bit
+    bool obsolete_3: 1;
+    bool reserved_5: 1;
+    bool wbus_16: 1; // Wide Bus bit 
+    bool sync: 1;
+    bool obsolete_4: 1;
+    bool reserved_6: 1;
+    bool cmdque: 1; // Command Management Model bit
+    bool vs_2: 1; // vendor specific bit
+    uint8_t vendor_identification[8];
+    uint8_t product_identification[16];
+    uint8_t product_revision_level[4];
```

Comment:
So for this field, we can also do `uint32_t` instead of a byte array. Should we set a standard of when we should just a byte array and when we should just use a primitive when both options work?

Reply:
Additionally, vendor_identification could be declared as a uint64_t

Reply:
That's a good catch! And I agree with having a standard for using an array vs a primitive. I would say that if we have a field > 64 bits, we should use an array and in all other cases, we should use the appropriate primitive (eg uint8_t, uint16_t etc)

Reply:
> Additionally, vendor_identification could be declared as a `uint64_t`

@harloff-g do you mean that we should declare `vendor_identification` as an array of `uint64_t`? @eyhhuang and I discussed that its better to have an array with elements of 1 byte each. What are your views on that?

Reply:
No, I just mean that vendor_identification doesn't have to be an array at all and can be declared as `uint64_t vendor_identification : 64;`

Reply:
Oh true! Sorry, I actually misread your previous comment to mean `product_identification`. That makes sense and I will update the field 👍